models/job_models.go

```diff
@@ -145,9 +145,36 @@ func (r *Job) AfterFind() error {
 	return nil
 }
 
+// JobLog stores chunk of logs from job.
+//reform:job_logs
+type JobLog struct {
+	JobID     string    `reform:"job_id"`
+	ChunkID   int       `reform:"chunk_id"`
+	Message   string    `reform:"message"`
+	LastChunk bool      `reform:"last_chunk"`
+	Time      time.Time `reform:"time"`
```

Comment:
Let's name it `CreatedAt`, same as it other models

Reply:
but it's not `CreatedAt` like in other models.

Reply:
But you set this field with the `BeforeInsert` hook. 

https://github.com/percona/pmm-managed/pull/822/files#diff-0f7b3f0c83c8bcffc7aec1f9d149124fa760b057781a6d22f054c5b0b000f2c3R159-R164

Reply:
u are right, my bad. Anyway, I'll remove this field